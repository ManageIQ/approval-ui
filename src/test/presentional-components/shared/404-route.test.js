import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import NoMatch from '../../../presentational-components/shared/404-route';
import { IntlProvider } from 'react-intl';

describe('<NoMatch />', () => {
  it('should render correctly', () => {
    expect(toJson(mount(<IntlProvider locale="en"><NoMatch /></IntlProvider>))).toMatchSnapshot();
  });
});
