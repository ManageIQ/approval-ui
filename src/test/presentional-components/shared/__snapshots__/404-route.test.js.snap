// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoMatch /> should render correctly 1`] = `
<IntlProvider
  defaultFormats={Object {}}
  defaultLocale="en"
  formats={Object {}}
  locale="en"
  messages={Object {}}
  onError={[Function]}
  textComponent={Symbol(react.fragment)}
>
  <NoMatch>
    <div>
      <h1>
        Page you are looking for does not exist.
      </h1>
    </div>
  </NoMatch>
</IntlProvider>
`;
